src/PythonMigrationViewExtension/Differ/SideBySideViewModel.cs

```diff
@@ -8,7 +8,7 @@ public class SideBySideViewModel : IDiffViewViewModel
         public SideBySideDiffModel DiffModel { get; set; }
         public DiffPaneModel AfterPane { get { return DiffModel.NewText; } }
         public DiffPaneModel BeforePane { get { return DiffModel.OldText; } }
-        private bool HasChanges { get { return DiffModel.NewText.HasDifferences | DiffModel.OldText.HasDifferences; } }
+        public bool HasChanges { get { return DiffModel.NewText.HasDifferences | DiffModel.OldText.HasDifferences; } }
```

Comment:
@mmisol @aparajit-pratap @mjkkirschner I dont recall why we specifically made this property private, but I would like to expose it on interface level and add analytics coverage so we know not only user clicked accept or Reject but also if there are any real code changes.